pkg/apis/options/legacy_options.go

```diff
@@ -97,17 +100,18 @@ func (l *LegacyUpstreams) convert() (Upstreams, error) {
 			upstream.Static = true
 			upstream.StaticCode = &responseCode
 
-			// These are not allowed to be empty and must be unique
+			// This is not allowed to be empty and must be unique
 			upstream.ID = upstreamString
-			upstream.Path = upstreamString
```

Comment:
Was this the root of the 404s reported?

Reply:
Indeed, it was mapping `static://xxx` to the path 🤦‍♂️